README.md

```diff
@@ -1 +1,90 @@
-# functional-harness
+## testcafe-functional-harness
+
+Runs functional tests in local browsers and on SauceLabs.
+
+#### What it does
+
+1. Starts the server with the test pages.
+2. Opens the specified browsers (locally or on SauceLabs).
+3. Runs mocha tests.
+4. Closes the browsers and stops the server after all tests are finished.
+
```

Comment:
The main goal - is to run TestCafe tests and handle failed reports, handle failed runs or check reports. We use mocha tests for this. Here the cases of using:
1) Run tests and check they are succeed
2) Run tests that should be failed with a specified error. Check the tests are failed. Then check the report contains an expected error
3) Run tests with some failed setting and check the run is failed with a specified error
4) Run successful tests and check the report is correct

The module allows to run this in specified browsers on saucelabs out-of-the-box. And it allows move out all stuff (saucelabs connecting, tested pages server, mocha customization) from the main `testcafe` repo.

@VasilyStrelyaev Could you please create a short description for `package.json` and a more detailed one for `readme.md` based on this? Feel free to ask if something is still unclear.